backends/arm/test/test_models.py

```diff
@@ -221,23 +180,7 @@ def forward(self, x):
             x = self.conv2d(x)
             return self.adaptive_avg_pool2d(x)
 
-    @register_test
-    class simple_softmax(torch.nn.Module):
-        inputs = {
-            TosaProfile.BI: (torch.ones(2, 3),),
-            TosaProfile.MI: (torch.ones(2, 3),),
-        }
-
-        permute_memory_to_nhwc = False
-
-        def __init__(self):
-            super().__init__()
-            self.softmax = torch.nn.Softmax(dim=1)
-
-        def forward(self, x):
-            return self.softmax(x)
-
-    @register_test
+    # @register_test
```

Comment:
Is this intentional?

Reply:
No, pushed a new patch reverting that change.